lib/Rx/Observable.php

```diff
@@ -112,54 +118,103 @@ public static function create(callable $subscribeAction)
     /**
      * Returns an observable sequence that produces a value after each period.
```

Comment:
A little more descriptive would be: "Returns an Observable that emits an infinite sequence of ascending integers starting at 0, with a constant interval of time of your choosing between emissions."